Scenes/MainLevel.tscn

```diff
@@ -13,7 +13,7 @@ shader = ExtResource( 8 )
 shader_param/scale = 2
 shader_param/threshold = 2.0
 shader_param/color = Color( 0, 0, 0, 1 )
-shader_param/enabled = true
+shader_param/enabled = false
```

Comment:
This is because of the thing you sent in telegram?

Reply:
No this mainly cos when the object got more details like the hands and legs then the outline is a bit problematic.

Reply:
oh ok